doc/source/examples/multiphysics/rayleigh-benard-convection/rayleigh-benard-convection.rst

```diff
@@ -0,0 +1,181 @@
+
+In this example, we simulate the Rayleigh-Benard convection problem at two Rayleigh numbers of 10000 and 25000. According to the literature [1, 2], we should see different numbers (2 and 3 vortices at Ra=10000 and 25000, respectively) of vortices in the fluid at these two Rayleigh numbers. Note that in this example, we change the Ra number by changing the thermal conduction coefficient of the fluid.
+
+.. note:: 
+    All the four boundary conditions are ``noslip``, and an external 
+    gravity field of :math:`-10` is applied in the :math:`y` direction.
+
+
+--------------
+Parameter file
+--------------
+
+Time integration is handled by a 1st order backward differentiation scheme 
+`(bdf1)`, for a :math:`10000` s simulation time with an initial 
+time step of :math:`0.01` second.
+
+.. note::   
+    This example uses an adaptive time-stepping method, where the 
+    time-step is modified during the simulation to keep the maximum value of the CFL condition below a given threshold (0.5 here). Using ``output control = time``, and ``output time = 25`` the simulation results are written every 25 s.
+
+.. note::   
+    Note that the heating process is slow, and the velocity magnitudes are small inside the fluid. Hence, we expect large time-steps and a long simulation.
+
+.. code-block:: text
+
+    # --------------------------------------------------
+    # Simulation Control
+    #---------------------------------------------------
+    subsection simulation control
+        set method                          = bdf1
+        set time end                        = 10000
+        set time step                       = 0.01
+        set adapt                           = true
+        set max cfl                         = 0.5
+        set stop tolerance                  = 1e-5
+        set adaptative time step scaling    = 1.3
+        set number mesh adapt               = 0
+        set output name                     = rayleigh-benard_convection
+        set output control                  = time
+        set output time                     = 25
+        set output path                     = ./output/     
+    end
+
+
+The ``multiphysics`` subsection enables to turn on `(true)` and off `(false)` the physics of interest. Here ``heat transfer``, ``buoyancy force``, and ``fluid dynamics`` are chosen.
+
+.. code-block:: text
+
+    #---------------------------------------------------
+    # Multiphysics
+    #---------------------------------------------------
+    subsection multiphysics
+        set buoyancy force      = true
+        set heat transfer       = true
+        set fluid dynamics      = true
+    end 
+    
+The ``source term`` subsection defines gravitational acceleration. The value of the gravitational acceleration in this example is selected to satisfy the desired values of Ra number.
```

Comment:
At line 53, it is stated that the Ra is set by changing the value of k. Here, one might think that it is set by changing the gravity magnitude. Maybe something like `` The gravity magnitude is set to -10 for both simulations. Thus, the Rayleigh number is controlled only by the thermal conduction coefficient for this example. `` could be added at line 53.

```suggestion
The ``source term`` subsection defines gravitational acceleration. The value of the gravitational acceleration in this example is set to -10.
```

Reply:
I've explained this issue in line 123 in detail, but thank you for your suggestion, this sentence is added to line 53 :)